From 23f1d700a37eced5fb33cc75b7b64ae5a3b0f0c9 Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Tue, 23 Jul 2013 18:35:38 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr

---
 neutronclient/version.py | 5 +----
 setup.py                 | 1 -
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/neutronclient/version.py b/neutronclient/version.py
index 9315671..f0b6efc 100644
--- a/neutronclient/version.py
+++ b/neutronclient/version.py
@@ -16,7 +16,4 @@
 # vim: tabstop=4 shiftwidth=4 softtabstop=4
 # @author: Carl Baldwin, Hewlett-Packard
 
-import pbr.version
-
-
-__version__ = pbr.version.VersionInfo('python-neutronclient').version_string()
+__version__ = "REDHATNEUTRONCLIENTVERSION"
diff --git a/setup.py b/setup.py
index 70c2b3f..afab729 100644
--- a/setup.py
+++ b/setup.py
@@ -18,5 +18,4 @@
 import setuptools
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
